EIPS/eip-712.md

```diff
@@ -386,7 +367,7 @@ The in-place implementation makes strong but reasonable assumptions on the memor
 
 Similarly, a straightforward implementation is sub-optimal for directed acyclic graphs. A simple recursion through the members can visit the same node twice. Memoization can optimize this.
 
-## Rationale for `domainSeparator`
+### Rationale for `domainSeparator`
 
 Since different domains have different needs, an extensible scheme is used where the DApp specifies a `EIP712Domain` struct type and an instance `eip712Domain` which it passes to the user-agent. The user-agent can then apply different verification measures depending on the fields that are there.
 
```

Comment:
(Referring to the line below this one. Couldn't comment directly on it.)

There is a reference and link to "[EIP-719](https://github.com/ethereum/EIPs/issues/719
)" which doesn't really exist. What would you suggest? @MicahZoltu Any suggestions being the author of that issue?

Reply:
Hm, while I'd recommend waiting for Micah's input, I think you can probably just remove this line. Since it isn't mentioned in the Specification section as a permitted field, this seems like a contradiction anyway.

Reply:
There is no EIP-719, so you can't/shouldn't link to it.  I would *love it* if someone would create an EIP-719 and champion it and push it through to final, but at the moment that isn't happening so removing the line seems like the best course of action.